app/services/game-overlay/index.ts

```diff
@@ -117,7 +117,15 @@ export class GameOverlayService extends PersistentStatefulService<GameOverlaySta
     this.assignCommonWindowOptions();
     const partition = this.userService.state.auth.partition;
     const chatWebPrefences = { ...this.commonWindowOptions.webPreferences, partition };
-    this.windows.recentEvents = new BrowserWindow({ ...this.commonWindowOptions, width: 600 });
+    this.windows.recentEvents = this.windowsService.createOneOffWindowForOverlay({
+      ...this.commonWindowOptions,
+      width: 600,
+      componentName: 'GameOverlayEventFeed',
+      queryParams: { gameOverlay: true },
+      webPreferences: { offscreen: true, nodeIntegration: true },
+      isFullScreen: true,
+      alwaysOnTop: true,
```

Comment:
do alwaysOnTop and isFullScreen need to be here? would have assumed the overlay plugin took care of that

Reply:
Probably not, will remove

Reply:
isFullScreen makes the window title bar go away so it is necessary after all, alwaysOnTop seems to happen anyway so that can be removed